src/main/webapp/app/exercises/quiz/manage/quiz-management.route.ts

```diff
@@ -23,16 +23,16 @@ export const quizManagementRoute: Routes = [
         canDeactivate: [PendingChangesGuard],
     },
     {
-        path: ':courseId/quiz-exercises/:exerciseId/re-evaluate',
-        component: QuizReEvaluateComponent,
+        path: ':courseId/quiz-exercises/export',
+        component: QuizExerciseExportComponent,
         data: {
-            authorities: [Authority.INSTRUCTOR, Authority.ADMIN],
+            authorities: [Authority.TA, Authority.EDITOR, Authority.INSTRUCTOR, Authority.ADMIN],
```

Comment:
The TA is not supposed to be able to export quiz exercises, so I guess we should remove the respective authority